corehq/ex-submodules/pillowtop/tests/test_elasticsearch.py

```diff
@@ -198,8 +198,8 @@ def _bad_es_getter():
             from elasticsearch import Elasticsearch
             return Elasticsearch(
                 [{
-                    'host': 'localhost',
-                    'port': '9000',  # bad port
+                    'host': settings.ELASTICSEARCH_HOST,
+                    'port': settings.ELASTICSEARCH_PORT - 2,  # bad port
```

Comment:
This test was failing for me because I have a service running on `localhost:9000`, and the ES service was running in boot2docker, which has a different IP address than localhost.